phrens/appease-server
# #167208866 setting up database

#### What does this PR do?
- setting up and configuring the database
- installing the necessary dependencies to run the database

#### Description of the tasks
- setting up and configuring the database
- installing the necessary dependencies to run the database

#### How should this be manually tested?
- git clone this [repo](https://github.com/phrens/appease-server/) `git clone https://github.com/phrens/appease-server/`
- git checkout to `ch-setup-database-167208866` branch `git checkout ch-setup-database-167208866`
- install the dependencies `yarn install`
- test the app `yarn dev`
- navigate to your terminal
- you should see
![Screenshot from 2019-07-13 20-12-18](https://user-images.githubusercontent.com/37577645/61174552-896a9e00-a5aa-11e9-9833-e9d20f6668bc.png)

#### Pivotal tracker styling
[167208866](https://www.pivotaltracker.com/story/show/167208866)
